tests/e2e/src/pages/base.page.ts

```diff
@@ -235,6 +239,7 @@ export class BasePage {
   }
 
   async returnElementByType(elementType: string, value: string) {
+    //element = undefined;
```

Comment:
Removed in the last commit

Reply:
it is not a comment it should be uncommented but not removed, otherwise we will reuse previous selector for undefined element an it will be always true

Reply:
Uncommented in latest commit